support-frontend/assets/components/subscriptionCheckouts/stripeForm/stripeProviderForCountry.tsx

```diff
@@ -33,6 +38,7 @@ function StripeProviderForCountry(props: PropTypes): JSX.Element {
 	);
 	useEffect(() => {
 		if (stripeObject === null) {
+			props.setStripePublicKey(stripeKey);
```

Comment:
I noticed that the field existed in the redux state, but wasn't getting populated for GW/GW gift/paper so it was still empty string, as they were using this `useState` mechanism instead.  I have just threaded it through to the redux to set it there as well now.